3.reports.sql

```diff
@@ -0,0 +1,24 @@
+use CostumeDB
+go
+
+-- 1. I need to know which costume is the most popular.
```

Comment:
AS: -2 For questions 1 and 2 it only asks to display the most popular one.

Reply:
fixed